test/screenshot/spec/fixture.scss

```diff
@@ -67,10 +67,19 @@ $test-layout-cell-grid-color: #ddd;
     // stylelint-enable selector-class-pattern
   }
 
-  .test-container--edge-fonts .test-font--flaky {
+  .test-container--edge-fonts .test-font--redact-all {
```

Comment:
I would assume there are places that were referencing this by its old name?

Reply:
Only #3413 references it for now